MAINTAINERS

```diff
@@ -76,6 +76,11 @@ R:	Linaro <op-tee@linaro.org>
 S:	Maintained
 F:	core/arch/arm/plat-poplar/
 
+Hisilicon hi3519Av100 family
```

Comment:
s/hi3519Av100/Hi3519AV100/

Reply:
yes, i will also update the same spelling error in README.MD